test/wallet-test-auctions.js

```diff
@@ -0,0 +1,184 @@
+/* eslint-env mocha */
```

Comment:
Do you mind moving this to the `auctions-test.js` file?

Reply:
The reason I didn't do this at first is because the current `auctions-test` seems to be testing consensus rules against `miner` and `chain`. That test uses `memwallet` which doesn't even [test for double opens](https://github.com/handshake-org/hsd/blob/master/lib/wallet/wallet.js#L1542-L1543).

Since this PR is focused on wallet behavior I thought it was best to start a new test file (other wallet auction functions can be added in here later). I could add `acutions-test` if you want, just wanted to throw out my motivation there...

Reply:
I see! Ok, can we call it `wallet-auctions-test.js` to stick to the current naming convention?

Reply:
👍